.github/workflows/test-env-deploy.yml

```diff
@@ -0,0 +1,63 @@
+name: TEST-ENV-DEPLOYMENT
+
+on:
+  pull_request:
+    types: [opened, reopened, edited, synchronize]
+    branches: ["*"]
```

Comment:
As asked and discussed internally, we need a check that it is not coming from a fork. If that's a pull request from a fork, then we shouldn't run (no GH Action failure, but a skip).

Reply:
By default secrets are not passed to forked PRs. This could be an easy way to ensure that we run it on blessed branches only.

Reply:
Solution: use `jobs.deploy.if: ...` https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idif